R/areachart.R

```diff
@@ -168,9 +169,9 @@ Area <- function(x,
     if (any(is.na(as.matrix(chart.matrix))))
         warning("Missing values have been interpolated or omitted.")
 
-    num.notNA <- colSums(!is.na(chart.matrix))
-    if (all(num.notNA < 2))
-        stop("Area is not defined for a series with only one data point.")
+    #num.notNA <- colSums(!is.na(chart.matrix))
+    #if (all(num.notNA < 2))
+    #    stop("Area is not defined for a series with only one data point.")
```

Comment:
These were commented out but not removed? I see the tests below have been updated to not error in this case. Likely because missing data is expectcted to produce an empty plot element?

Reply:
Thanks, I've removed these now